tests/test_wave_01.py

```diff
@@ -118,13 +118,10 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    assert MOVIE_TITLE_1 in updated_data["watched"][0]["title"]
```

Comment:
This passes the assertion for sure, but it's a bit dangerous to use `in` here because the check is for a substring within a string, e.g. `"food" in "food"` which would return `True`. So this test would also pass if `MOVIE_TITLE_1` were `"It Came from the Stack Tracer"`, which it shouldn't. You could just test for equality in these instances:
```python
    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
    assert updated_data["watched"][0]["genre"] == GENRE_1
```
and so on.